antenna-documentation/src/site/markdown/docker.md

```diff
@@ -101,4 +103,23 @@ If you have a project that needs to be scanned by Antenna, do not simple mount t
 `antenna-related-files` folder, because then the project's source code itself would not be mounted
 and a proper scan could not be executed.
 
+### Using files storing properties
```

Comment:
Minor nit: The term "storing properties" is a bit misleading IMHO, as the file defines properties or environment variables that can be referenced in the configuration. So should the caption better be "Using files defining properties" or "variables"?

Reply:
I agree, it makes more sense